src/components/lab/ListItemView/ListItemView.scss

```diff
@@ -0,0 +1,285 @@
+@use '../../variables';
+@use '../../../../styles/mixins';
+
+$block: '.#{variables.$ns}lab-list-item-view';
+
+/* ListItemView CSS API */
+#{$block} {
+    /* Sizes */
+    --_--g-list-item-view-height: var(--g-list-item-view-height, #{variables.$m-height});
+    --_--g-list-item-view-border-radius: var(
+        --g-list-item-view-border-radius,
+        var(--g-border-radius-m)
+    );
+    --_--g-list-item-view-padding-inline: var(
+        --g-list-item-view-padding-inline,
+        var(--g-spacing-2)
+    );
+    --_--g-list-item-view-padding-block: var(--g-list-item-view-padding-block, var(--g-spacing-1));
```

Comment:
Let's use single prop `--g-list-item-view-padding` for consistency with other components